README.md

```diff
@@ -88,6 +88,23 @@ runtime library, using the `MSVC_STATIC_RUNTIME` setting:
 
     $ cmake -DMSVC_STATIC_RUNTIME=ON -DBUILD_SHARED_LIBS=OFF ..
 
+## Fuzzing libmaxminddb locally
```

Comment:
Could we put these in a separate doc? e.g. README.fuzzing.md perhaps. Mostly because I think most users won't need to see this and I'd like to keep the main README simpler if possible.

Reply:
Done;